qualitag/src/main/resources/application.properties

```diff
@@ -11,4 +11,7 @@ spring.data.mongodb.uri=mongodb://localhost:27017/qualitag
 token.secret.key=48a868a4042f634ac04a117f00a87202131dd7c46c4b32c4acb3edc5e15f4511
 
 # JWT expiration is 1 hour
-token.expiration=3600000
\ No newline at end of file
+token.expiration=3600000
+
+# JWT password reset token is 15 minutes
+token.password.expiration=15
```

Comment:
Interesting! maybe we can communicate the token lifespan to the user via the mail, such as: "The link will last 15 minutes". (Nice to have addition)